functions/controllers/application/index.js

```diff
@@ -100,7 +100,7 @@ application.get("/checkApp", jwtCheck, hasReadApp, async (req, res) => {
     res.status(200).send({ code: 200, status: appStatus, exists: true, message: "Document Found" });
   } catch (error) {
     if (error.message === "No Document") {
-      res.status(200).send({ code: 200, status: "No Document", exists: false, message: "No Document" });
+      res.status(200).send({ code: 404, status: "No Document", exists: false, message: "No Document" });
     } else {
```

Comment:
Be sure to update `.status()` to equal `code` in the response body if you like to use a 404. Also, I'm not 100% sure if sending a 404 is the best in this case. If an app doesn't exist for a user, it's not necessarily a bad request, it just means the user hasn't submitted a document, so I would think this should be a 2XX response code.

Reply:
OK that makes more sense. I thought `res.body.status` would be our custom status messages and different from the HTTP response codes. 

And yes, this should be a 200 response code.